oem.json

```diff
@@ -1,4 +1,7149 @@
+        "web": [
+            "https://en.wikipedia.org/wiki/Voice_over_LTE"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.indexsearch",
+        "description": "Index search works without it.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.indexsearch.observer",
+        "description": "Index search works without it.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.lbs",
+        "label": "HwLBService",
+        "description": "Location Service? After removal, location still works, so no one knows what this app does.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.mmitest",
+        "label": "MMITest",
+        "description": "Hidden hardware tests. Safe to remove.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.motionservice",
+        "label": "Gesture Service",
+        "description": "Useless gestures available in settings.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.nearby",
+        "label": "HwNearby",
+        "description": "Needed for Huawei Share (com.huawei.android.instantshare).",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.omacp",
+        "description": "Used for provisioning APN settings to devices via SMS. I think you shouldnt touch it if you want sms messages.",
+        "removal": "caution"
+    },
+    {
+        "id": "com.huawei.rcsserviceapplication",
+        "label": "Huawei RCS",
+        "description": "Hidden RCS Chats with 49 permissions.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.recsys",
+        "label": "HwIntelligentRecSystem",
+        "description": "is a smart app that provides personalized services on HiBoard? My phone dont have Hiboard. Safe to remove.",
+        "web": [
+            "https://consumer.huawei.com/en/support/content/en-us00434788/"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.screenrecorder",
+        "label": "screen recorder",
+        "description": "Not sure if it's an app, but I can't find it on my phone. Safe to remove.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.huawei.systemmanager",
+        "label": "System manager app",
+        "description": "DO NOT REMOVE THIS. You lose phone functionality.",
+        "removal": "unsafe"
+    },
+    {
+        "id": "com.huawei.systemserver",
+        "description": "It depends on (com.huawei.systemmanager). DO NOT REMOVE THIS. You lose phone functionality.",
+        "removal": "unsafe"
+    },
+    {
+        "id": "com.huawei.trustcircle",
+        "description": "Its very unknown app but remove it if you have removed (com.huawei.hwid), (com.huawei.appmarket). \nThis is what I found on this app files: Huawei Mobile Services?, installing,updating apps AppGallery.\nSo it looks like you will not have access to huawei hwid account if you remove it.",
```

Comment:
Avoid second person. Use passive form for them. Example:

The app's usage is unknown, but it can be removed if apps such as com.huawei.hwid, com.huawei.appmarket are also removed. This is what I found while exploring the APK: Huawei Mobile Services?,  managing installation and updates of AppGallery.

And move the rests to the `warning` field:

Huawei HWID account may be inaccessible without this app